### ISSUE TITLE
feature request: metriki support metric label

### ISSUE THREAD

**Issue by SSebo** [sunng87/metriki#94]
maybe macro use like this:
`timer!("grpc.bind_tag", "direction" => direction.to_string(), "namespace" => namespace.to_string());`

**Comment by sunng87** [comment]
Yes, tag has been a critical missing feature for this library (and dropwizard metrics). Will you be interested in a pull request for this feature?

Internally we may need to create timer for each distinct tag value pairs.

**Comment by SSebo** [comment]
I found some crate generate a hash by `metric_name` and a set of tags. and store the metric value in that hash key related store

> Internally we may need to create timer for each distinct tag value pairs.
what is used for `timer` ?

**Comment by sunng87** [comment]
I think we will need timer for each tags value tuples, for example when `direction` is `up`/`down`, `namespace` is `a`/`b`, there will be 4 timers for each possible combination

```
grpc.bind_tag{direction="up", "namespace"="a"} 
grpc.bind_tag{direction="up", "namespace"="b"} 
grpc.bind_tag{direction="down", "namespace"="a"} 
grpc.bind_tag{direction="down", "namespace"="b"} 
```

**Comment by sunng87** [comment]
implemented in #95
